src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/IValueProviderFactory.cs

```diff
@@ -10,10 +10,10 @@ public interface IValueProviderFactory
         /// <summary>
         /// Gets a <see cref="IValueProvider"/> with values from the current request.
         /// </summary>
-        /// <param name="context">The <see cref="ValueProviderFactoryContext"/>.</param>
+        /// <param name="context">The <see cref="ActionContext"/>.</param>
         /// <returns>
         /// A <see cref="Task"/> that when completed will yield a <see cref="IValueProvider"/> instance or <c>null</c>.
         /// </returns>
-        Task<IValueProvider> GetValueProviderAsync(ValueProviderFactoryContext context);
+        Task<IValueProvider> GetValueProviderAsync(ActionContext context);
```

Comment:
This was a holdover from previous iterations where modelbinding was a separate package.